src/main/java/de/retest/recheck/ignore/InverseFilter.java

```diff
@@ -0,0 +1,23 @@
+package de.retest.recheck.ignore;
+
+import de.retest.recheck.ui.descriptors.Element;
+import de.retest.recheck.ui.diff.AttributeDifference;
+
+public class InverseFilter implements Filter {
```

Comment:
Puh, that's a though decision. While Git allows this, there are three things two consider:

1. This breaks our rule "filters are only additive", now they can be conflicts I guess.
2. Isn't blacklist testing something we discourage?
3. There should be a way to also do this from `recheck.ignore`.

Reply:
Removed